draft-ietf-quic-http.md

```diff
@@ -343,20 +348,30 @@ certificate for the origin before considering it authoritative. Clients MUST NOT
 assume that an HTTP/3 endpoint is authoritative for other origins without an
 explicit signal.
 
+Clients SHOULD NOT open more than one HTTP/3 connection to a given host and port
+pair, where the host is derived from a URI, a selected alternative service
+{{!ALTSVC}}, or a configured proxy.  A client MAY open multiple connections to
+the same IP address and UDP port using different transport or TLS configurations
+but SHOULD avoid creating multiple connections with the same configuration.
+
 Prior to making requests for an origin whose scheme is not "https," the client
 MUST ensure the server is willing to serve that scheme.  If the client intends
 to make requests for an origin whose scheme is "http", this means that it MUST
 obtain a valid `http-opportunistic` response for the origin as described in
 {{!RFC8164}} prior to making any such requests.  Other schemes might define
 other mechanisms.
 
+Servers are encouraged to maintain open connections for as long as possible but
+are permitted to terminate idle connections if necessary.  When either endpoint
+chooses to close the QUIC connection, the terminating endpoint SHOULD first send
+a GOAWAY frame ({{connection-shutdown}}) so that both endpoints can reliably
+determine whether previously sent frames have been processed and gracefully
+complete or terminate any necessary remaining tasks.
```

Comment:
The text here requires both endpoints can send GOAWAY, which I think means a dependency on https://github.com/quicwg/base-drafts/pull/3129

Reply:
Possibly, but not a hard one.  GOAWAY permits the endpoints to agree on state, so I'm content not to order them.